src/components/Modal/index.js

```diff
@@ -67,8 +96,11 @@ const Modal = ({ open, setOpen, query }) => {
     }
   }
 
-  const nextLinkText = query.nextLinkText
-  const prevLinkText = query.prevLinkText
+  const onClickHandler = (e) => {
+    setToggleDrawer(!toggleDrawer)
+
+    setContactPerson(e.currentTarget.childNodes[1].childNodes[1].innerText)
```

Comment:
what? If that works fine but I'm pretty sure you dont need to go that deep in the object

Reply:
If so thats totally fine